stix2validator/test/attack_pattern_tests.py

```diff
@@ -26,7 +26,7 @@ class AttackPatternTestCases(ValidatorTest):
     valid_attack_pattern = json.loads(VALID_ATTACK_PATTERN)
 
     def test_wellformed_attack_pattern(self):
-        results = validate_string(VALID_ATTACK_PATTERN, self.options)
+        results, = validate_string(VALID_ATTACK_PATTERN, self.options)
```

Comment:
Hmm, I didn't know that notation worked on lists as well as tuples. `validate_string(...)[0]` seems more readable to me, but don't feel like you need to change it.

Since `validate_string` returns a list now, the elevator may need some changes depending on which `validate_*` function it's using.

Reply:
`validate_string(...)[0]` seems more standard to me than the list unpacking syntax. I'm also not sure if I like `validate` (and by extension, `validate_string`), returning a list when you only pass one object. On one hand, I like the consistency, but it seems like it would be confusing to me.